docs/guides/react-components/install-plain-java.md

```diff
@@ -8,8 +8,169 @@ For this installation, you must have:
 - OpenJDK 21+ locally installed
 - Camunda `8.6.0-alpha2` or later
 
+### Download and configure Elasticsearch
+
+1. Download [Elasticsearch 8.9.2](https://www.elastic.co/downloads/past-releases/elasticsearch-8-9-2).
+2. Disable Elasticsearch's security packages by setting the `xpack.security.*` configuration options to `false` in `ELASTICSEARCH_HOME/config/elasticsearch.yml`.
+3. Start Elasticsearch by running `ELASTICSEARCH_HOME/bin/elasticsearch` (or `ELASTICSEARCH_HOME\bin\elasticsearch.bat` on Windows).
+
+### Download and configure Camunda
+
 1. Download the [latest release artifact](https://github.com/camunda/camunda/releases), starting with [8.6.0-alpha2](https://github.com/camunda/camunda/releases/tag/8.6.0-alpha2).
-2. Download [Elasticsearch 8.9.2](https://www.elastic.co/downloads/past-releases/elasticsearch-8-9-2).
-3. For non-production cases, disable Elasticsearch's security packages by setting the `xpack.security.*` configuration options to `false` in `ELASTICSEARCH_HOME/config/elasticsearch.yml`.
-4. Start Elasticsearch by running `ELASTICSEARCH_HOME/bin/elasticsearch` (or `ELASTICSEARCH_HOME\bin\elasticsearch.bat` on Windows).
-5. To start Camunda, run `bin/camunda` (or `bin\camunda.bat` on Windows).
+2. Modify the `application.yaml` to include the Elasticsearch exporter by adding the following to line 91:
+
+```
+    exporters:
+      elasticsearch:
+        className: io.camunda.zeebe.exporter.ElasticsearchExporter
+        args:
+          url: http://localhost:9200
+          index:
+            prefix: zeebe-record
+```
+
+:::note
+Spacing is important! Ensure four spaces and proper nesting of configuration.
```

Comment:
```suggestion
Spacing is important! Indent the `exporters` element four spaces to properly nest the configuration.
```

"Ensure four spaces" threw me off because the example uses two-space indentation....but I think you meant that the `exporters` element itself is indented four spaces.

Reply:
If there is a consistent sibling next to it, then I think that makes sense! But I don't have a strong opinion either way.

Reply:
After further review, I think this can be aided by my comment on the previous block -- specifying the _logical_ location of the configuration block, instead of physical.